dashboards/jenkins/jenkins-prometheus.json

```diff
@@ -59,219 +64,425 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "(vm_memory_total_max{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}} - vm_memory_total_used{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}}) / vm_memory_total_max{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}} * 100.0"
+                  "prometheusQuery": "(vm_memory_total_max{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}} - vm_memory_total_used{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}}) / vm_memory_total_max{job=\"jenkins-metrics\",${Cluster},${Location},${Namespace}} * 100.0",
```

Comment:
I noticed in a test environment we have, the job here is named `jenkins` instead of `jenkins-metrics` so this chart was blank. I'm not honestly sure if there's a standard name. What do you think of a looser regex match like `job=~"jenkins.*"` which at least catches both of our setups?

Reply:
I'm not sure it's a standard, but in the documentation the PodMonitoring resource name is `jenkins`, whereas in the environment on our end, we named it `jenkins-metrics`. I think that with the documentation having `jenkins` in the example yaml, it's somewhat safe to assume that the word jenkins will be in the name somewhere. Does opting for `job=~".*jenkins.*"` make more sense in your opinion?

Reply:
@johnbryan I've pushed up my suggestion, let me know if you think it makes sense. Hope all is well!

Reply:
Looks great, thanks for the fix! (and apologies for slow replies here, I had some vacation time recently)